File: bill.api/app/database/entity/BcUser.ts
import { PrimaryGeneratedColumn} from "typeorm";
import {BaseEntity} from "../BaseEntity";
import {Column,Entity} from "../decorator";

@Entity()
export class BcUser extends BaseEntity {
    @PrimaryGeneratedColumn("uuid")
    id: string;

    @Column()
    name: string;

    @Column()
    loginName: string;

    @Column()
    loginPassword: string;

    @Column()
    pic: string;

    @Column()
    bussWx: string;
}

File: com.joey.bill/src/main/java/com/joey/bill/model/entity/BcUser.java
package com.joey.bill.model.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Table;

@Entity
@Table(name = "bc_user")
public class BcUser extends BaseEntity{
    @Column
    private String name;

    @Column(name = "login_name")
    private String loginName;

    @Column(name = "login_password")
    private String loginPassword;

    @Column
    private String pic;
}
